dev-perl/Net-IDN-Encode/files/2.500.0-use-uvchr_to_utf8_flags-instead-of-uvuni_to_utf8_fla.patch

```diff
@@ -0,0 +1,41 @@
+https://bugs.gentoo.org/912440
```

Comment:
I'll merge this with a new revision - it only failed to build w/ tinderbox because of Clang 16.

Implicit function declarations need to go straight to hell :(